### ISSUE TITLE
Gradle plugin should be deployed to plugins.gradle.org

### ISSUE THREAD

**Issue by hastebrot** [Kotlin/kotlinx.serialization#256]
This will allow to use (`build.gradle.kts` script):

```kt
plugins {
    val kotlinVersion = "1.3.0"
    kotlin("jvm") version kotlinVersion
    kotlin("serialization") version kotlinVersion
}
``` 

instead of

```kt
buildscript {
    val kotlinVersion = "1.3.0"

    repositories {
        jcenter()
    }

    dependencies {
        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
        classpath("org.jetbrains.kotlin:kotlin-serialization:$kotlinVersion")
    }
}

apply {
    plugin("kotlin")
    plugin("kotlinx-serialization")
}
```

For Kotlin 1.3.0 there are multiple plugins already deployed to https://plugins.gradle.org/search?term=org.jetbrains.kotlin

**Comment by sandwwraith** [comment]
https://youtrack.jetbrains.com/issue/KT-27612

**Comment by sandwwraith** [comment]
Fixed in Kotlin 1.3.50
